### ISSUE TITLE
DZAwareness

### ISSUE THREAD

**Issue by Lemonadez** [berbb/PortAIO#339]
I pick Morgana and every it load in-game, it just hang.
I tested it again (alt+f4) and reconnect. Same it just freeze (hang) after the game load.

I had to disabled DZAwareness by picking Ahri again and uncheck on settings. 

Playing on Summoner Rift 5x5 (Normal).

**Comment by Lemonadez** [comment]
And for Ahri BTW


Exception occured! EloBuddy might crash!

```
Type: System.NullReferenceException
Message: Object reference not set to an instance of an object.

Stracktrace:
   at OneKeyToWin_AIO_Sebby.Core.MissileReturn.CalculateReturnPos()
   at OneKeyToWin_AIO_Sebby.Core.MissileReturn.Game_OnGameUpdate(EventArgs args)
   at EloBuddy.Game.OnGameUpdateNative()

```

**Comment by KiriharaSenpai** [comment]
i think berb pushed a fix for it but i personally fix it with disable q helper and missile return in menu

**Comment by Muse30** [comment]
this error is pretty old I think it was fixed (Ahri) maybe try to change the prediction

**Comment by Lemonadez** [comment]
Ye this was a problem too on Sivir/Ahri back on 6.11 (It was fixed). 
After 6.12 iupdate the problem returned.

And Now it was fixed on 6.12 16 hrs ago from this post.
https://github.com/berbb/PortAIO/commit/c68f7207a84d979018776f6b1cc8356b98eca816

Close n Fixed.
